src/pytom_tm/entry_points.py

```diff
@@ -831,7 +758,7 @@ def match_template(argv=None):
 
     # combine ctf values to ctf_params list of dicts
     ctf_params = None
-    if args.defocus_file is not None:
+    if args.defocus is not None:
```

Comment:
This whole function is untested, would it make sense to add tests to this entry point in this PR as well?

Reply:
yea, could do it. Do you have any examples of proper entry point testing with all arguments and stuff?

Reply:
It is quite trivial, you just pass it a list of arguments that are split on spaces
so `['-t', template_path, '-v', tomogram_path]` etc or you can just write the string with command options (not the entry point name itself) and use `commands.split()`

Reply:
This is added